data/xml/A83.xml

```diff
@@ -130,7 +130,7 @@
 </paper>
 
  <paper id="1009">
- <title>DISTINGUISHING FACT FROM OPINlON AND EVENTS FROM META-EVENTS</title>
+ <title><fixed-case>DISTINGUISHING</fixed-case> <fixed-case>FACT</fixed-case> <fixed-case>FROM</fixed-case> OPINlON AND EVENTS <fixed-case>FROM</fixed-case> META-<fixed-case>EVENTS</fixed-case></title>
```

Comment:
Something has gone wrong here. I thought there was an exception made for a title that was in all caps.

Reply:
I see a lowercase L (or is it a 1?) in "opinion"...

Reply:
Doh. I can fix it